ddtrace/internal/constants.py

```diff
@@ -16,3 +16,8 @@
 W3C_TRACESTATE_ORIGIN_KEY = "o"
 W3C_TRACESTATE_SAMPLING_PRIORITY_KEY = "s"
 DEFAULT_SERVICE_NAME = "unnamed_python_service"
+
+TELEMETRY_TRACER = "tracers"
+TELEMETRY_APPSEC = "appsec"
```

Comment:
A slightly more descriptive naming convention on these constants would help readers understand their intended purpose. `TELEMETRY_TAG_TRACER`, maybe?

Reply:
You're right, I think the correct name would be `TELEMETRY_NAMESPACE_TAG_TRACER`